lib/kube/proxy/forwarder.go

```diff
@@ -430,8 +448,19 @@ type handlerWithAuthFuncStd func(ctx *authContext, w http.ResponseWriter, r *htt
 
 // authenticate function authenticates request
 func (f *Forwarder) authenticate(req *http.Request) (*authContext, error) {
-	const accessDeniedMsg = "[00] access denied"
+	ctx, span := f.cfg.tracer.Start(
+		req.Context(),
+		"kube/Forwarder.authenticate",
```

Comment:
Is this naming convention used elsewhere in the project already?

I grepped the project for "tracer.start" and it seems that we'd typically set this to `forwarder/authenticate`. OTOH `kube/Forwarder.authenticate` is more specific and you immediately know that it's related to Kubernetes.